### PR TITLE
Merge master

### PR DESCRIPTION
**Pull request by avoliva**
- [ ] 🎉 Feature Enhancement
- [ ] 🐛 Bug Fix
- [ ] 📖 Documentation Update
- [ ] ❇️ Other

### CODE REVIEW

**Review by harshad-symbl — APPROVED** [review]

**Review by amritesh-singh — APPROVED** [review]

File: docs/web-sdk/muting-and-unmuting-connected-device.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/muting-and-unmuting-connected-device.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/overview.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/overview.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/passing-a-custom-sourceNode.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/passing-a-custom-sourceNode.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/passing-custom-ondevicechange-handler.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/passing-custom-ondevicechange-handler.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/reconnecting-real-time.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/reconnecting-real-time.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/stopping-realtime-connection.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/stopping-realtime-connection.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/subscribe-to-realtime.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/subscribe-to-realtime.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/transcribing-live-audio-through-microphone.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/transcribing-live-audio-through-microphone.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

File: docs/web-sdk/web-sdk-reference.md [hunk 9]
```diff
@@ -0,0 +1,369 @@
+---
```
Comment (pema-s) [inline docs/web-sdk/web-sdk-reference.md:1]: This file should be deleted. It contains Web SDK details which we are not pushing right now (as mentioned earlier).

**Review by pema-s — APPROVED** [review]

**Pull request merged**